lib/addDefaults.js

```diff
@@ -347,6 +347,42 @@ function addDefaultTechnique(gltf) {
     }
 }
 
+function isTechniqueShared(gltf, techniqueId) {
+    var users = 0;
+    var materials = gltf.materials;
+    for (var materialId in materials) {
+        if (materials.hasOwnProperty(materialId)) {
+            var material = materials[materialId];
+            if (material.technique === techniqueId) {
+                ++users;
```

Comment:
Why not return early, i.e., `return true;` when this is `> 1`, then just always `return false;` below.